### PR TITLE
Convert SyntheticWheelEvent-test.js to createRoot

### PR DESCRIPTION
**Pull request by mattcarrollcode**

### CODE REVIEW

File: packages/react-dom/src/events/__tests__/SyntheticWheelEvent-test.js [hunk 1]
```diff
@@ -83,7 +92,7 @@ describe('SyntheticWheelEvent', () => {
     expect(events[1].deltaY).toBe(-50);
   });
 
-  it('should be able to `preventDefault` and `stopPropagation`', () => {
+  it('should be able to `preventDefault` and `stopPropagation`', async () => {
```
Comment (rickhanlonii) [inline packages/react-dom/src/events/__tests__/SyntheticWheelEvent-test.js:95]: Can add an `expect.assertions(3)` to the end of this test

**Review by rickhanlonii — APPROVED** [review]
Looks good!

The test failure looks like an unrelated flaky test, I re-ran the job.

File: packages/react-dom/src/events/__tests__/SyntheticWheelEvent-test.js (hunk 1 above)
Comment (mattcarrollcode) [reply packages/react-dom/src/events/__tests__/SyntheticWheelEvent-test.js:95]: Shouldn't the expected number of assertions be 5? 2x for each dispatchEvent call (4 total) and one expect events to be "2".

Comment (rickhanlonii) [reply packages/react-dom/src/events/__tests__/SyntheticWheelEvent-test.js:95]: Yeah I counted wrong

**Pull request merged**